src/database/static_methods.rs

```diff
@@ -62,6 +62,62 @@ impl Database {
         Ok((ret.iter().all(|&x| x), event_ids))
     }
 
+    pub(crate) fn delete_event_helper(
+        connection: &mut rusqlite::Connection,
+        index_writer: &mut IndexWriter,
+        event_id: EventId,
+        undeleted_events: &mut Vec<EventId>,
+    ) -> Result<bool> {
+        let transaction = connection.transaction()?;
+
+        transaction.execute("DELETE from events WHERE event_id == ?1", &[&event_id])?;
+        transaction.execute(
+            "INSERT OR IGNORE INTO undeleted_events (event_id) VALUES (?1)",
```

Comment:
Hmm, does `undeleted_events` mean "events to be deleted"? If so, "undeleted" is confusing to me as I would assumed it means "restored" or "brought back from the trash".

If my assumption is correct, maybe use `pending_deletion_events` or something?

Reply:
It is events to be deleted.

Reply:
+1 tp `pending_deletion_events`